.travis/before_install.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+if [ "$TYPE" != 'style' ]; then
+  # shellcheck disable=SC1090
+  . "$TRAVIS_BUILD_DIR/.travis/$TRAVIS_OS_NAME.$TARGET_OS.before_install.sh"
```

Comment:
How do you want me to remove sourcing ? I could do:

`chmod +x file && ./file` or `bash file`, no idea what's best.